tests/memory_tests/src/memory_tests_helper/memory_counter.cpp

```diff
@@ -72,32 +72,34 @@ namespace MemoryTest {
 
 #else
 
-/// Parses number from provided string
-    static int parseLine(std::string line) {
-        std::string res = "";
-        for (auto c: line)
-            if (isdigit(c))
-                res += c;
-        if (res.empty())
-            // If number wasn't found return -1
-            return -1;
-        return std::stoi(res);
-    }
-
-    size_t getSystemDataByName(char *name) {
-        FILE *file = fopen("/proc/self/status", "r");
+    size_t getSystemDataByName(char* name) {
```

Comment:
In general memory tests could reuse implementation from the utils library

Reply:
Yes, but historically it was appeared in stress tests first 😄